src/main/java/frc/robot/subsystems/GripperSubsystem.java

```diff
@@ -16,10 +16,12 @@ public GripperSubsystem() {}
   private final CANSparkMax gripmot = new CANSparkMax(Constants.Gripper, MotorType.kBrushless);
 
   public void rotateoutGripper() {
+    System.out.println("Running Claw");
     gripmot.set(0.5);
   }
 
   public void rotateinGripper() {
+    System.out.println("Running Claw");
```

Comment:
Do we still need these print statements?

Reply:
its helpful to keep that way we can tell immediately if there is something wrong with the CAN